api/src/domain/license-status/searchBusinessAddressesForMatch.test.ts

```diff
@@ -0,0 +1,107 @@
+import { generateLicenseSearchAddress } from "@shared/test";
+import { searchBusinessAddressesForMatch } from "./searchBusinessAddressesForMatch";
+
+describe("searchBusinessAddressesForMatch", () => {
+  const userInputtedAddress = generateLicenseSearchAddress({
+    addressLine1: "Main St",
+    addressLine2: "Apt 1",
+    zipCode: "12345",
+  });
+  const matchedAddress1 = generateLicenseSearchAddress({
+    addressLine1: "Main St",
+    addressLine2: "Apt 1",
+    zipCode: "12345",
+  });
+  const matchedAddress2 = generateLicenseSearchAddress({
+    addressLine1: "Main St",
+    addressLine2: "Apt 1",
+    zipCode: "12345",
+  });
+  const randomBusinessWithRandomAddresses1 = {
+    businessId1234: [generateLicenseSearchAddress({}), generateLicenseSearchAddress({})],
+  };
+  const randomBusinessWithRandomAddresses2 = {
+    businessId2345: [generateLicenseSearchAddress({}), generateLicenseSearchAddress({})],
+  };
+  const businessWithNoAddresses = {
+    businessId3456: [],
+  };
+
+  describe("single business ids", () => {
+    it("returns correct business id when it has two matching addresses", () => {
+      const businessId1 = { matchingId: [matchedAddress1, matchedAddress2] };
```

Comment:
[sand] It might be easier to read if this variable was named something else, because this object is not a "businessId", it's a "businessIdToAddressesMapping" and that's important.

Also, this is tiny, but I think it's useful to put `"matchingId"` in quotes here in this object, which helps trigger the brain that this is a test string and not an actual key in a defined type (if that makes sense)